R/validate.R

```diff
@@ -0,0 +1,56 @@
+#' Validate a package source
+#'
+#' @param pkg Top-level directory of the package to validate
+#' @param ... Additional argument passed to `devtools::build()`
+#'
+#' @importFrom devtools build
+#' @importFrom rmarkdown render
+#'
+#' @export
+vt_validated_build <- function(pkg = ".", ...) {
+
+  tryCatch({
+    file.copy(from = paste0(c(pkg, "inst", "validation"),sep = .Platform$file.sep),
+              to = paste0(c(pkg, "vignettes"), sep = .Platform$file.sep), recursive = TRUE)
+
+
+    render(paste0(c(pkg, "inst", "validation", "validation.Rmd"), sep = .Platform$file.sep),
+           output_dir = paste0(c(pkg, "inst"), sep = .Platform$file.sep))
+
+    inform("Validation Complete", class = "vt.validation")
```

Comment:
Validation Report Generated, validation is not complete unless the org accepts the act of generation as validation